templates/release-dist-packages.yml

```diff
@@ -0,0 +1,61 @@
+spec:
+  inputs:
+    stage:
+      description: The pipeline stage where to trigger the distribution packages pipeline
+      default: publish
+    runner:
+      description: GitLab CI runner tag
+      default: hetzner-amd-beefy
+    mender-dist-packages-branch:
+      description: Branch to trigger in mender-dist-packages repository
+      default: master
+    package:
+      description: The name of the package to build, must be uppercase. See mender-dist-packages variables
+    test-mender-dist-packages:
+      description: Run tests
+      type: boolean
+      default: true
+    publish-mender-dist-packages:
+      description: Publish packages
+      type: boolean
```

Comment:
I don't know if this will work. The variables are passed to the triggered pipeline, from where are interpreted as strings. We'll see 🧪